package.json

```diff
@@ -1281,6 +1292,49 @@
           "type": "integer",
           "default": 1,
           "description": "Number of header rows to assume. Set to 0 to disable.`"
+        },
+        "latex-workshop.formattedPaste.image.method": {
+          "type": "string",
+          "enum": [
+            "leave",
+            "copy",
+            "move"
+          ],
+          "enumDescriptions": [
+            "reference the image where it is",
+            "copy image to target location",
+            "move image to target location"
+          ],
+          "default": "leave",
+          "description": "When pasting the path to an image, determine how it is treated. With copy and move 'target location' is graphicsdir or cwd"
+        },
+        "latex-workshop.formattedPaste.image.template": {
+          "type": [
+            "string",
+            "array"
+          ],
+          "default": "\\includegraphics{${imageFilePath}}",
+          "description": "The template to use for pasting the image (use an array for multiline strings), use ${imageFilePath} to represent the image file. Can also use ${imageFileNameWithoutExt}"
+        },
+        "latex-workshop.formattedPaste.image.defaultName": {
```

Comment:
Just make it the default and remove the config. Also, I'm a bit worried about the size of moment, especially when we can play with `new Date()` for date and `new Date().toLocaleTimeString()` for time.

Reply:
See 51c026ab907634c1401d7fe18a1de974f5c1bc90